extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -280,6 +281,25 @@ const OHIFCornerstoneViewport = React.memo(props => {
       viewportOptions.viewportType = STACK;
     }
 
+    if (needsRerendering) {
+      displaySets.forEach(displaySet => {
+        const viewportInfo = CornerstoneViewportService.getViewportInfoByIndex(
+          viewportIndex
+        );
+
+        if (viewportInfo.hasDisplaySet(displaySet.displaySetInstanceUID)) {
+          const viewportData = viewportInfo.getViewportData();
+          const currentViewportData = (viewportData) ? viewportData : { viewportType: STACK };
```

Comment:
Consider replacing the two lines with a single line. Not a big deal, just something to consider. :)

```const currentViewportData = viewportInfo.getViewportData() ?? { viewportType: STACK };```